### ISSUE TITLE
[ERROR] Username is not set

### ISSUE THREAD

**Issue by xavierzhao** [Digital-Sapphire/PyUpdater-SCP-Plugin#2]
```
D:\sources\python\gc>pyupdater settings --plugin scp
Enter username
[DEFAULT] -> gctest
Press Enter To Use Default
-->
You entered gctest, is this correct?
[N/y]?y
Enter host
[DEFAULT] -> 192.168.1.189
Press Enter To Use Default
-->
You entered 192.168.1.189, is this correct?
[N/y]?y
Enter remote directory
[DEFAULT] -> /var/www/html/gcbuild
Press Enter To Use Default
-->
You entered /var/www/html/gcbuild, is this correct?
[N/y]?y
[INFO] Saving Config
[INFO] Config saved
[INFO] Wrote client config
[INFO] Settings update complete

D:\sources\python\gc>pyupdater upload --service scp
[ERROR] Username is not set; please report this issue on https://github.com/JMSwag/PyUpdater/issues
```

**Comment by xavierzhao** [comment]
@JMSwag Here's where it went wrong and how to fix it, thanks & best regards.

**Comment by JMSwag** [comment]
I'll get this resolved & push v2.0 to pypi this weekend.

**Comment by JMSwag** [comment]
@xavierzhao PyUpdater 2.0.3 & PyUpdater-SCP-plugin 3.0.5 have been uploaded to pypi. Please rerun pyupdater settings --plugins scp once you update both packages.
